DataLayer/Wearable/src/main/AndroidManifest.xml

```diff
@@ -68,5 +68,6 @@
                 <category android:name="android.intent.category.DEFAULT" />
             </intent-filter>
         </activity>
+        <activity android:name="androidx.activity.ComponentActivity" />
```

Comment:
Rather than adding this to main.  I think you can add a test dependency on androidx.compose.ui:ui-test-manifest

Reply:
removed, thanks!